lib/db.js

```diff
@@ -13,6 +13,15 @@ const data = {
         "```Looks like something went wrong. Need not worry. Here are some logs since when the bot was not responding as expected.```\n```---------```\n```🧐 Command:``` *{commandName}*\n```😎 From Me?:``` *{fromMe}*\n```🗣️ Was a reply?:``` *{isReply}*\n```👥 In a group?``` *{isGroup}*\n```📥 In Inbox?``` *{isPm}*\n```📸 Command with image?``` *{isImage}*\n```🕺🏻 Is Bot group admin?``` *{isBotGroupAdmin}*\n```📈 Was Sender group admin?``` *{isSenderGroupAdmin}*\n```🫂 Was sender sudo?``` *{isSenderSudo}*\n```⚠️ Error:``` \n*{err}*\n```---------```\n_To figure out what exactly went wrong, please report/raise the issue on our support chat at_ https://chat.whatsapp.com/Gc8gNMoue2uHyd6xhFwvEP",
     SUCCESSFUL_CONNECTION: "*BotsApp successfuly integrated.*\n```Bot is currently working in``` *{worktype}* ```mode.```\n```For more information regarding Bot working and permissions check out:``` \n🔗https://github.com/BotsAppOfficial/BotsApp."
   },
+  abl: {
+    DESCRIPTION: "Module to blacklist a person or a chat.",
+    EXTENDED_DESCRIPTION:
+          "```Add new people to a group by entering their mobile number as per the format mentioned below.\n    1. XXXXXXXXXX\n    2. YYXXXXXXXXXX ()\n\nFor example -\n``` *.add 9861212121*",
+    PM_ACKNOWLEDGEMENT: "```{user} banned from using Bot in all chats.```",
```

Comment:
```suggestion
    PM_ACKNOWLEDGEMENT: "```{user} banned from using the bot in all chats.```",
```